src/Components/Loading/Loading.scss

```diff
@@ -0,0 +1,84 @@
+.loading {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 5px;
+
+  .dots {
+    width: 5em;
+    height: 5em;
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    grid-template-columns: repeat(3, 1fr);
+    grid-auto-flow: row;
+
+    .dot1 {
+      width: 0.1em;
+      height: 0.1em;
+      border: 3px solid black;
+      border-radius: 100%;
+      background-color: black;
+      animation: growAndShrink infinite 2.5s;
+      @keyframes growAndShrink {
```

Comment:
This looks very cool! You've really gone above and beyond the assessment brief here! 🎉 🎊 

One small note: I think the keyframes are the same for each dot, so is it possible to define `growAndShrink` in only one place rather than repeat it for each dot? I am not an expert in this area but I believe it might be possible to dry this up, especially since you have installed sass.